tests/algorithms/algorithm_settings.py

```diff
@@ -145,21 +160,30 @@ def get_alg_kwargs(alg_cls: Type[Algorithm]) -> Dict[str, Any]:
 def get_alg_model(alg_cls: Type[Algorithm]) -> ComposerModel:
     """Return an instance of the model for an algorithm."""
     settings = _get_alg_settings(alg_cls)['model']
+    if isinstance(settings, ComposerModel):
```

Comment:
Can you remove this check? See above comments re: just putting the constructor in the dictionary.